config/manifests/bases/metallb-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: metallb-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: An operator for deploying MetalLB on a kubernetes cluster.
+  displayName: Metallb Operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - metallb-operator
+  links:
+  - name: Metallb Operator
+    url: https://github.com/metallb/metallb-operator
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat
```

Comment:
same here

Reply:
I think this should be one of the upstream developers who develop and maintain this operator, and not the Red Hat support address for the upstream operator.  If you're not sure who to put, you're free to use me: Russell Bryant <rbryant@redhat.com>.

Reply:
though I'm less involved in the operator than all of you, so I'd say decide among yourselves who to put, but I'm ok as a backup :)